contributors/devel/development.md

```diff
@@ -159,17 +159,14 @@ brew install coreutils ed findutils gawk gnu-sed gnu-tar grep make jq
 ```
```

Comment:
this list should probably be updated. I have `bash coreutils gnu-sed gnu-tar`, I'm not sure you need the rest anymore.

system make and jq seem to be working fine (jq is more recently added to macOS base), though I might've missed something.

Reply:
(we can/should probably leave that for a different PR, and double check the current scripts)

Reply:
I agree it seems to work fine with the system `make` and `jq`, though I do wonder whether there may be future requirements that depend on features in more recent versions?

I've often had issues with the system versions of software on macOS.

`jq` seems to be relatively recent at `jq-1.6-159-apple-gcff5336-dirty` on my system (homebrew is `1.7.1`), though `make` is pretty old at `GNU Make 3.81` on my system, with the latest in homebrew being `4.4.1`.

Reply:
I'll try to invest some time in figuring out which of the other dependencies listed are needed and raise as a separate PR as you suggest 👍

Reply:
If we require a specific version in a script we typically add a check for it and prompt the user there at least. I'm on the fence about asking people to install them again here preemptively

Reply:
I'll leave the list for now and try to review that as a separate PR 👍